src/trixi.h

```diff
@@ -32,6 +32,16 @@ int trixi_nvariables(int handle);
 double trixi_calculate_dt(int handle);
 void trixi_load_cell_averages(double * data, int handle);
 
+// T8code
+#if !defined(T8_CMESH_H)
+typedef struct t8_cmesh *t8_cmesh_t;
+#endif
+#if !defined(T8_FOREST_GENERAL_H)
+typedef struct t8_forest *t8_forest_t;
+#endif
```

Comment:
```suggestion
#if !defined(T8_H) && !defined(T8_CMESH_H)
typedef struct t8_cmesh *t8_cmesh_t;
#endif
#if !defined(T8_H) && !defined(T8_FOREST_GENERAL_H)
typedef struct t8_forest *t8_forest_t;
#endif
```
I'd also include the `T8_H` header guard just to be on the safe side. They might rename the file where these typedefs live, but they most likely will not rename `t8.h`.

Reply:
Ok! I now understand why you initially suggested this!